### PR TITLE
Bump version to 1.6.2

### PR DESCRIPTION
**Pull request by zhonghui12**
*Issue #, if available:*

*Description of changes:*
Bump version to 1.6.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.6.2
```
Comment (hossain-rayhan) [inline CHANGELOG.md:3]: Aren't we adding the kinesis_stream plugin changes. We merged teo different PRs.

Comment (hossain-rayhan) [reply CHANGELOG.md:3]: Sorry. I got it wrong! I thought it was our main image repo.

**Review by hossain-rayhan — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+* Enhancement - Add validation to stop accepting both of LogStream and LogStreamPrefix together (#190)
```
Comment (PettitWesley) [inline CHANGELOG.md]: nit: LogStreamPrefix and LogStream are not the exact names of the config options

**Review by PettitWesley — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (zhonghui12) [reply CHANGELOG.md]: I will rename it

**Pull request merged**